spec/features/recipes/show_spec.rb

```diff
@@ -0,0 +1,50 @@
+require "rails_helper"
+
+RSpec.describe "the recipes show" do
+  before(:each) do
+    @recipe = Recipe.create!(name: "Salted Ground Beef", complexity: 1, genre: "Carnivore")
+    @ingredient_1 = @recipe.ingredients.create!(name: "Ground Beef", cost: 2)
+    @ingredient_2 = @recipe.ingredients.create!(name: "Salt", cost: 4)
+  end
+
+  it "lists all the recipes attributes" do
+    visit "/recipes/#{@recipe.id}"
+    
+    expect(page).to have_content(@recipe.name)
+    expect(page).to have_content(@recipe.complexity)
+    expect(page).to have_content(@recipe.genre)
```

Comment:
Same idea here! These assertions would be more robust if they verified that these attributes appeared next to the right labels